imagelab/DisplayImageTest.java

```diff
@@ -0,0 +1,57 @@
+package imagelab;
+
+import org.junit.Before;
+import org.junit.After;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.*;
+
+public class DisplayImageTest {
+    /**
+     * Sets up the test fixture.
+     * (Called before every test case method.)
+     */
+    @Before
+    public void setUp() {
+       ImageLab labOne = new ImageLab();
```

Comment:
Please remove this. (This appears to create and initialize an unused local variable in a `setUp` method.)